helm-service/chart/templates/_helpers.tpl

```diff
@@ -36,9 +36,7 @@ Common labels
 {{- define "helm-service.labels" -}}
 helm.sh/chart: {{ include "helm-service.chart" . }}
 {{ include "helm-service.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
+app.kubernetes.io/version: {{ include "service.tag" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.AppVersion) }}
```

Comment:
Is `"service.tag"` available here? I only found the definition in the core installer

Reply:
ok, thx for clarifying :)